cloudevents/extensions/attrschema-schema.yaml

```diff
@@ -0,0 +1,97 @@
+title: "Attrschema meta-schema"
+$schema: "http://json-schema.org/draft-07/schema#"
+$id: "https://cloudevents.io/attrschema/2022-10/schema#"
+allOf:
+  - { "$ref": "http://json-schema.org/draft-07/schema#" }
```

Comment:
Looks like this is using draft-07 of JSON Schema. The current stable version is 2020-12. Is there a particular reason for not updating to 2020-12?

Reply:
no, fixing now